ZenPacks/zenoss/LinuxMonitor/migrate/DevCpuUtilizationGraph.py

```diff
@@ -0,0 +1,34 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2016, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+
+from Products.ZenModel.ZenPack import ZenPackMigration
+from Products.ZenModel.migrate.Migrate import Version
+
+import logging
+log = logging.getLogger("zen.migrate")
+
+
+class DevCpuUtilizationGraph(ZenPackMigration):
+
+    version = Version(2, 0, 0)
+
+    def migrate(self, pack):
+        log.info('Fixing the Device template')
+        try:
+            path = '/zport/dmd/Devices/Server/SSH/Linux/rrdTemplates/Device'
+            template = pack.dmd.unrestrictedTraverse(path)
+            graph = template.graphDefs._getOb('CPU Utilization')
+        except Exception:
+            log.info('Unable to find CPU Utilization graph for Device')
```

Comment:
I'd drop this log message. There's not exceptional about an exception occurring here. It could happen either because the monitoring template doesn't exist, or the graph we're trying to remove not existing. Neither condition would be at all interesting to the user.